### PR TITLE
YAML file for the MIT Supercloud Dataset

### PR DESCRIPTION
**Pull request by ssamsis**
*Issue #, if available:*

*Description of changes:*
Adding YAML file for the MIT Supercloud Dataset. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: datasets/dcc.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Citation: Samsi, Siddharth, Weiss, Matthew, Bestor, David, et al. "The MIT Supercloud Dataset." 2021 IEEE High Performance Extreme Computing Conference (HPEC). IEEE, 2021. 
+Resources:
+  - Description: The MIT Supercloud Dataset
+    ARN: arn:aws:s3:::mit-supercloud-dataset
+    Region: us-west-2
+    Type: S3 Bucket
+DataAtWork:
+  Tutorials:
+    - Title: Introduction to the MIT Supercloud Dataset
+      URL: https://nbviewer.org/github/MIT-AI-Accelerator/MIT-Supercloud-Dataset/blob/main/notebooks/Intro-to-MIT-Supercloud-Dataset.ipynb
```
Comment (jflasher) [inline datasets/dcc.yaml:23]: Looks to be an issue with this notebook.

File: datasets/dcc.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      AuthorName: Matthew Weiss
+    - Title: Generating the labelled dataset
+      URL: https://nbviewer.org/github/MIT-AI-Accelerator/workload-classification/blob/main/notebooks/Generating-the-labelled-dataset.ipynb
```
Comment (jflasher) [inline datasets/dcc.yaml:26]: This notebook is failing to load as well.

File: datasets/dcc.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+Name: The MIT Supercloud Dataset
+Description: Collection of parsed datacenter logs and time series data of hardware utilization from the MIT Supercloud system. 
+Documentation: https://github.com/MIT-AI-Accelerator/datacenter-challenge
+Contact: mit-dcc@mit.edu
+ManagedBy: Siddharth Samsi
+UpdateFrequency: Data will be updated annually
+Tags:
+  - datacenter
```
Comment (jflasher) [inline datasets/dcc.yaml:8]: Please add `aws-pds` as a tag as well, thanks!

**Review by jflasher — CHANGES_REQUESTED** [review]
Hi @ssamsis, left a few comments inline, thanks!

**Comment by jflasher** [comment]
Hi @ssamsis wanted to check back in here, thanks!

**Comment by ssamsis** [comment]
The YAML file has been updated and issues with the notebooks have been resolved.

**Comment by jflasher** [comment]
Hi @ssamsis, looks like https://nbviewer.org/github/MIT-AI-Accelerator/workload-classification/blob/main/notebooks/Generating-the-labelled-dataset.ipynb is still throwing an error.

**Comment by ssamsis** [comment]
The notebook URLs have been update and should work now. Thank you!

**Review by jflasher — APPROVED** [review]

**Comment by jflasher** [comment]
@ssamsis this should be good to go. It's having an issue with a conflict on the tags.yaml. I'll look to resolve that in the next dew days by starting a new, clean PR. Thanks!

**Pull request closed without merging**